projects/plugins/jetpack/extensions/blocks/voice-to-content/voice-to-content.php

```diff
@@ -18,10 +18,17 @@
  * registration if we need to.
  */
 function register_block() {
-	Blocks::jetpack_register_block(
-		__DIR__,
-		array( 'render_callback' => __NAMESPACE__ . '\load_assets' )
-	);
+	/**
+	 * Register the block only if we are on an A8C P2 site.
+	 */
+	if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+		if ( function_exists( 'wpcom_is_automattic_p2_site' ) && wpcom_is_automattic_p2_site() ) {
+			Blocks::jetpack_register_block(
+				__DIR__,
+				array( 'render_callback' => __NAMESPACE__ . '\load_assets' )
+			);
+		}
+	}
```

Comment:
Might be good to leave a comment stating we'll have to add a check for `apply_filters( 'jetpack_ai_enabled', true )` and `( new Host() )->is_wpcom_simple() || ! ( new Status() )->is_offline_mode()` if we get this feature out of P2 later, to be in line with the Assistant.

Reply:
Indeed! Going to update the PR.